core/test/unit/models/post_spec.js

```diff
@@ -365,6 +365,24 @@ describe('Unit: models/post: uses database (@TODO: fix me)', function () {
         });
     });
 
+    describe('toJSON', function () {
```

Comment:
@kirrg001 let me know if this looks good and will merge right after :cowboy_hat_face:

Reply:
This describe uses the db layer, I'd suggest to put none db access tests to the first describe?

Reply:
Moved couple more tests over with the toJSON one

Reply:
👍 perfecto. I think this is ready to merge 👍